keyboards/Boardsource-4x12-blok/kb.py

```diff
@@ -1,25 +1,25 @@
-# auto generated
 import board
 
 from kmk.kmk_keyboard import KMKKeyboard as _KMKKeyboard
+from kmk.quickpin.pro_micro.boardsource_blok import pinout as pins
 from kmk.scanners import DiodeOrientation
 
 
 class KMKKeyboard(_KMKKeyboard):
-    row_pins = (board.RX, board.TX, board.SDA, board.SCL)
+    row_pins = (board.pins[1], pins[0], board.pins[4], board.pins[5])
```

Comment:
Is this the right way to address `pins[]`? Other files do not call as `board.pins[]`

Reply:
https://github.com/KMKfw/kmk_firmware/blob/master/docs/quickpin.md

Reply:
Seems like my sed failed. Corrected, thank you.